packages/dmn-js-shared/package.json

```diff
@@ -20,6 +20,7 @@
     "inferno-test-utils": "~5.0.5"
   },
   "dependencies": {
+    "camunda-dmn-moddle": "^1.0.0",
```

Comment:
Generally we'd want to avoid depending on camunda-dmn-moddle. Before it was removed from dmn-moddle we were implicitly depending on it, which is to be avoided. It is [required by one feature only](https://github.com/bpmn-io/dmn-js/blob/develop/packages/dmn-js-decision-table/src/features/decision-table-head/editor/components/InputEditor.js). In the future we should move that feature to an extension.

Reply:
Agreed.

Reply:
I created an issue for that https://github.com/bpmn-io/dmn-js/issues/455